rate.go

```diff
@@ -26,6 +26,7 @@ type Rate struct {
 	DeliveryDate           *time.Time `json:"delivery_date,omitempty"`
 	DeliveryDateGuaranteed bool       `json:"delivery_date_guaranteed,omitempty"`
 	EstDeliveryDays        int        `json:"est_delivery_dats,omitempty"`
+	BillingType            string     `json:"billing_type,omitempty"`
```

Comment:
Does the smartrate strict need this too?

Reply:
No, it doesn't. This question was answered in Java PR: https://github.com/EasyPost/easypost-java/pull/141#discussion_r892859981